components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/store/SessionDataStore.java

```diff
@@ -348,7 +348,7 @@ public SessionContextDO getSessionContextData(String key, String type) {
             }
         } catch (ClassNotFoundException | IOException | SQLException | SessionSerializerException |
                 IdentityApplicationManagementException e) {
-            log.error("Error while retrieving session data", e);
+            log.debug("Error while retrieving session data", e);
```

Comment:
let's check if debug is enabled before adding debug. As that is the general format we use.